### PR TITLE
[sql gateway]Implement flink native

### PR DESCRIPTION
**Pull request by GOODBOY008**
## What changes were proposed in this pull request

Issue Number: close #xxx 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)

### CODE REVIEW

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/FlinkCluster.java [hunk 1]
```diff
@@ -213,4 +213,9 @@ public Map<String, Object> getProperties() {
     }
     return map;
   }
+
+  public String getGatewayAddress() {
+    // todo get gateway address
+    return this.address.replace("8081", "8083");
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/FlinkCluster.java]: This gateway address cannot be handled in this way, We will discuss how to solve this problem。

**Comment by wolfboys** [comment]
This is a great new feature. Could you provide some more detail about this PR?

**Comment by wolfboys** [comment]
Thanks for your contribution. if this PR is ready, You can call me to review it.

**Comment by GOODBOY008** [comment]
@wolfboys Please code review , if you are available.

File: streampark-console/streampark-console-service/pom.xml [hunk 2]
```diff
@@ -400,6 +416,12 @@
             <artifactId>force-shading</artifactId>
             <scope>provided</scope>
         </dependency>
+
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <version>${mysql.version}</version>
```
Comment (Zzm0809) [inline streampark-console/streampark-console-service/pom.xml]: double import

**Review by wolfboys — APPROVED** [review]

**Pull request merged**